crates/sncast/tests/e2e/account/import.rs

```diff
@@ -683,3 +683,97 @@ pub async fn test_invalid_address_computation() {
         error: Computed address {computed_address} does not match the provided address 0x123. Please ensure that the provided salt, class hash, and account type are correct.
     "});
 }
+
+#[tokio::test]
+pub async fn test_happy_case_unnamed_import() {
+    let tempdir = tempdir().expect("Unable to create a temporary directory");
+    let accounts_file = "accounts.json";
+
+    let import_args = vec![
+        "--accounts-file",
+        accounts_file,
+        "account",
+        "import",
+        "--url",
+        URL,
+        "--address",
+        "0x123",
+        "--private-key",
+        "0x456",
+        "--class-hash",
+        DEVNET_OZ_CLASS_HASH_CAIRO_0,
+        "--type",
+        "oz",
+    ];
+
+    let delete_args = vec![
+        "--accounts-file",
+        &accounts_file,
+        "account",
+        "delete",
+        "--name",
+        "account-2",
+        "--network",
+        "alpha-sepolia",
+    ];
+
+    for _ in 0..3 {
+        let snapbox = runner(&import_args).current_dir(tempdir.path());
+        snapbox.assert().stdout_matches(indoc! {r"
+        command: account import
+        add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
+    "});
+    }
+
+    let snapbox = runner(&delete_args).current_dir(tempdir.path());
+    snapbox.assert().success().stdout_matches(indoc! {r"
+        command: account delete
+        result: Account successfully removed
+    "});
+
```

Comment:
Let's check the accounts file content here. It should only have `account-1` and `account-3`.